docs/dagster-university/pages/dagster-dbt/lesson-7/5-preparing-for-a-successful-run.md

```diff
@@ -61,22 +61,47 @@ Because we’re still using a DuckDB-backed database, our `type` will also be `d
 
 ---
 
+## Adding a prod target to DbtProject
+
+Next, we need to update the `DbtProject` object in `dagster_university/assets/dbt.py` to specify what profile to target. To optimize the developer experience, let’s use an environment variable to specify the profile to target.
+
+1. In the `.env` file, define an environment variable named `DBT_TARGET` and set it to `dev`:
+
+   ```python
+   DBT_TARGET=dev
+   ```
+
+2. Next, import the `os` module at the top of the `dbt.py` file so the environment variable is accessible:
+
+   ```python
+   import os
+   ```
+
+3. Finally, scroll to the initialization of the `DbtProject` object, and use the new environment variable to access the profile to target. This should be on or around line 11:
+
+```python
+dbt_project = DbtProject(
+   project_dir=Path(__file__).joinpath("..", "..", "..", "analytics").resolve(),
+   target=os.getenv("DBT_TARGET")
+)
+```
+
+---
+
 ## Adding a prod target to deploy.yml
 
-Next, we need to update the dbt commands in the `.github/workflows/deploy.yml` file to target the new `prod` profile. This will ensure that dbt uses the correct connection details when the GitHub Action runs as part of our Dagster+ deployment.
+After that, we need to update the dbt commands in the `.github/workflows/deploy.yml` file to target the new `prod` profile. This will ensure that dbt uses the correct connection details when the GitHub Action runs as part of our Dagster+ deployment.
 
-Open the file, scroll to the dbt step you added, and add `-- target prod` after the `dbt parse` command. This command should be on or around line 52:
+Open the file, scroll to the environment variable sections, and set an environment variable named `DBT_TARGET` to `prod`. This command should be on or around line 12:
```

Comment:
```suggestion
Open the file, scroll to the environment variable section, and set an environment variable named `DBT_TARGET` to `prod`. This should be on or around line 12:
```

I'm assuming there's only one section. I'm also removing `command`, as I think it's a leftover from the previous version of this page.